content/docs/introducing-jsx.md

```diff
@@ -162,10 +162,10 @@ const element = React.createElement(
 );
 ```
 
-`React.createElement()` performs a few checks to help you write bug-free code but essentially it creates an object like this:
+`React.createElement()` ასრულებს რამდენიმე შემოწმებას რომ დაგეხმაროთ ბაგებისაგან თავისუფალი კოდის დაწერაში, მაგრამ არსებითად ის უბრალოდ ქმნის ამგვარ ობიექტებს:
 
 ```js
-// Note: this structure is simplified
+// შენიშვნა: ქვემოთმოცემული ობიექტის სტრუქტურა გამარტივებულია
```

Comment:
ქვემოთ მოცემული ცალკე არ უნდა ეწეროს?

Reply:
როგორც მახსოვს, გაზედსართავებული კომპოზიტი სიტყვაა ეს და, ვგონებ, ერთად უნდა იწერებოდეს.

Reply:
ახლა ვნახე და, ena.ge-ის თუ დავუჯერებთ, ცალ-ცალკე უნდა დაიწეროს: 'ქვემოთ მოცემული'

Reply:
resolved as of 35a3d34de37e265b949294074b9fea45374d8a7b